.github/workflows/e2e_genpolicy.yml

```diff
@@ -0,0 +1,65 @@
+name: genpolicy regression tests
+
+on:
+  schedule:
+    - cron: "16 6 * * 6" # 6:16 on Saturdays
```

Comment:
Should we combine this with the getdents regression test? For example, put the tests in one workflow, using one job per test?

Reply:
Done.